bluesky/plan_patterns.py

```diff
@@ -62,6 +62,102 @@ def spiral(x_motor, y_motor, x_start, y_start, x_range, y_range, dr, nth, *,
     cyc += cycler(y_motor, y_points)
     return cyc
 
+        
+def spiral_square_pattern(x_motor, y_motor, x_centre, y_centre, x_range, y_range, x_num,y_num):
+    '''Square spiral scan, centered around (x_start, y_start)
+    
+    Parameters
+    ----------
+    x_motor : object, optional
+        any 'setable' object (motor, temp controller, etc.)
+    y_motor : object, optional
+        any 'setable' object (motor, temp controller, etc.)
+    x_centre : float
+        x center
+    y_centre : float
+        y center
+    x_range : float
+        x width of spiral
+    y_range : float
+        y width of spiral
+    x_num : float
+        number of x axis points
+    y_num : float (must be even if x_num is even and must be odd if x_num is odd, if not it is increased by 1 to ensure this)
+        number of y axis points
+   
+   Returns
```

Comment:
This is not aligned (missing just one space) which will break the sphinx rendering. It's fussy!